### DISCUSSION TITLE
Why were code blocks removed as a MessageOption?

### DISCUSSION THREAD

**Discussion by Glazelf** [discordjs/discord.js#6001] [Q&A] [answered]
In 985d4d6a438fc9123264f6a1c600f34fccc1825f code blocks got removed as a MessageOption.
The result of this change is that instead of:
![Old code example](https://i.imgur.com/sDs2iq4.png)
You now seemingly have to use:
![New code example](https://i.imgur.com/X4gFhPj.png)
The new solution seems a lot uglier.  

TLDR: I don't understand why the `code` messageOption was deprecated. Is there any reason to this or plans for an alternative?

**Comment by kyranet** [comment, marked as answer]
See https://github.com/discordjs/discord.js/pull/5918, as well as https://github.com/discordjs/discord.js/pull/5976 for the alternative.

**Reply by Glazelf** [reply]
I missed those, thanks for the heads up!
